src/js/juju-cards-v1.0.0.js

```diff
@@ -60,6 +60,12 @@ let jujuCards = () => {
     }
 
     let addLink = `${demoDomain}/?deploy-target=${getImageID(id)}`;
+    let deployTitle = 'Deploy with Juju';
+
+    if (typeof(card.dataset.dd) !== 'undefined') {
+      addLink = `${demoDomain}/?dd=${getImageID(id)}`;
+      deployTitle = 'Direct Deploy with JAAS';
```

Comment:
I've actually been asked to change it to simply "Deploy with JAAS" so going to update it to that.